### PR TITLE
Refactor/awt 63 third person steps

### PR DESCRIPTION
**Pull request by d4vidmc**
It was set with "And opens the popover from header title"

### CODE REVIEW

File: src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java [hunk 1]
```diff
@@ -32,30 +32,30 @@
     /**
      * Delete through link option.
      */
-    public void clickOnDeleteProjectLink() {
+    public void clickOnDeleteWorkspaceProjectLink() {
```
Comment (88mary256) [inline src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java]: please restore before name

File: src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java [hunk 2]
```diff
@@ -32,30 +32,30 @@
...
         action.scrollToElement(deleteLink);
         action.click(deleteLink);
     }
 
     /**
      * Confirm deletion.
      */
-    public void clickOnDeleteButton() {
+    public void clickOnConfirmWorkspaceProjectDeleteButton() {
```
Comment (88mary256) [inline src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java]: please restore before name

File: src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java (hunk 1 above)
Comment (d4vidmc) [reply src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java]: This method was moved to respective Workspace/Project settings page

File: src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java (hunk 2 above)
Comment (d4vidmc) [reply src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java]: change to `clickOnDeleteButton` on next commit

**Review by 88mary256 — APPROVED** [review]

**Pull request merged**